docs/io/unified_image.rst

```diff
@@ -6,16 +6,22 @@ Image Data
 Reading and writing CCD image data in the unified I/O interface is supported
 though the :ref:`CCDData class <ccddata>` using FITS file format:
 
-.. doctest-skip::
-
-    >>> # Read CCD image
+.. testsetup::
     >>> from astropy.nddata import CCDData
-    >>> ccd = CCDData.read('image.fits')
+    >>> ccd = CCDData([1, 2, 3], unit='adu')
+    >>> ccd.write('image.fits')
+
+>>> # Read CCD image
```

Comment:
Indentation for clarity... (though different from test set up!? Or with `::` above?)